test/structured_metadata_spec.js

```diff
@@ -0,0 +1,160 @@
+const expect = require("expect.js");
+const cloudinary = require("../cloudinary");
+const helper = require("./spechelper");
```

Comment:
Need to finish the tests for upload, update, and explicit. This means adding functionality to the uploader. You can take a look at the implementation updateMetadata in Java